pkg/deploy/assets/rp-production-predeploy.json

```diff
@@ -157,13 +161,102 @@
                             "direction": "Inbound"
                         },
                         "name": "deny_in_gateway"
+                    },
+                    {
+                        "properties": {
+                            "protocol": "Tcp",
+                            "sourcePortRange": "*",
+                            "destinationPortRange": "444",
+                            "sourceAddressPrefixes": "[parameters('rpNsgPortalSourceAddressPrefixes')]",
+                            "destinationAddressPrefix": "*",
+                            "access": "Allow",
+                            "priority": 130,
```

Comment:
This priority will clash with the GenevaActions rule. It needs to be lower than 145 (the DENY rule) and not clash with another. 142 would be a good choice.